turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -507,16 +495,6 @@ impl BroadcastRun for StandardBroadcastRun {
     }
 }
 
-fn should_chain_merkle_shreds(slot: Slot, cluster_type: ClusterType) -> bool {
-    match cluster_type {
-        ClusterType::Development => true,
-        ClusterType::Devnet => true,
-        // Roll out chained Merkle shreds to ~53% of mainnet slots.
-        ClusterType::MainnetBeta => slot % 19 < 10,
```

Comment:
for the backports, can we simply set this to `true`? removal fine to ride master

Reply:
I don't see the point of keeping unused code around.
In particular, keeping more discrepancy between branches would introduce more risk if more changes are backported and they hit merge conflicts or something is missed out because the branches are different.

Reply:
i don't see the point of taking extra changes to stable

Reply:
avoiding unnecessary discrepancy between branches:
 
> In particular, keeping more discrepancy between branches would introduce more risk if more changes are backported and they hit merge conflicts or something is missed out because the branches are different.

Reply:
we don't backport feature gate removals. this is the same class of change

Reply:
Backporting a feature gate removal will break ledger tool for slots across epoch-boundary that the feature was activated. And a feature gate removal does not need to be backported anyways. But we need to backport this change; so not sure how it makes them in the same class.

And I think keeping the backported change the same as the master branch is a better approach for the reasons mentioned earlier above. Unnecessarily causing them to digress is only asking for problems.